dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/TaskExecutionContextFactory.java

```diff
@@ -108,8 +112,10 @@ public TaskExecutionContext createTaskExecutionContext(TaskExecutionContextCreat
 
         AbstractParameters baseParam =
                 TaskPluginManager.parseTaskParameters(taskInstance.getTaskType(), taskInstance.getTaskParams());
+
         Map<String, Property> propertyMap =
-                curingParamsService.paramParsingPreparation(taskInstance, baseParam, workflowInstance);
+                curingParamsService.paramParsingPreparation(taskInstance, baseParam, workflowInstance,
+                        project.getName(), workflowDefinition.getName());
```

Comment:
Directly transport project and workflowDefinition is better, then we can avoid to change this method in the future once we want to inject new parameter.